source/components/passing-properties-to-a-component.md

```diff
@@ -1,97 +1,88 @@
-By default a component does not have access to any properties normally
-available in the template in which it is used (sometimes referred to
-as the template `scope`)
+Components only have access to attributes passed to them.
```

Comment:
I like terse, but maybe expand a tiny bit? Remember you removed the whole index.

```
Components are isolated from their surroundings, so any data that the component needs has to be passed in.
```

?